lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -675,6 +676,17 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  createTreeWalker(root, whatToShow, filter) {
+    if (!("_ownerDocument" in root)) throw new TypeError("First argument to createTreeWalker must be a Node");
+    if (whatToShow === undefined) whatToShow = 0xFFFFFFFF;
```

Comment:
Cool, so these two lines should be unnecessary. Let us know if removing them breaks something, @garycourt.